tests/cases/compiler/commentsdoNotEmitComments.ts

```diff
@@ -88,3 +88,8 @@ module m1 {
 
 /// this is x
 declare var x;
+
+
+/** const enum member value comment (generated by TS) */

```

Comment:
can we have this as a separate test instead of combining with existing case?
Thanks.